solutions/src/main/kotlin/de/thermondo/solutions/basics/Basics06Solution.kt

```diff
@@ -0,0 +1,14 @@
+package de.thermondo.solutions.basics
+
+/**
+ * Create a class named Basics6Solution
+ *
+ * Define member variables:
+ * name: String
+ * age: Int
+ * and return the data of the two people while writing unit tests.
+ */
+class Basics06Solution(name: String, age: Int) {
+    val name = name // The variables that are part of the class are called as properties.
+    var age = age
```

Comment:
```suggestion
    val age = age
```

The general rule of thumb is to make everything a `val` 🧑‍🏫